terra/admin.py

```diff
@@ -89,6 +100,7 @@ class TravelRequestAdmin(admin.ModelAdmin):
         "funded",
         "closed",
         "allocations_total",
+        "approved_total",
```

Comment:
@aprigge With this added column, even my widescreen monitor doesn't display all columns without horizontal scrolling, and that scrollbar appears only at the bottom, below all data, so there was no visual clue that there are 3 currency columns until I saw that.

Maybe not much can be done about that, but a suggestion: several columns could have shorter names in the `/admin/terra/travelrequest/` display. Perhaps:
* Days out of office -> Days Out
* Administrative -> Admin
* For the 3 currency columns, remove Total as there are no other figures so Total is not needed